packages/discovery-react-components/package.json

```diff
@@ -71,5 +71,9 @@
       "!src/**/__stories__/**",
       "!src/**/*.stories.*"
     ]
+  },
+  "devDependencies": {
```

Comment:
do we use these in discovery-react-components?

Reply:
Unsure why, but I had to install these in order to get Storybook to run/build. Unsure why they weren't needed before.